JobSeek/JobSeek.Api/Services/Implementation/EmployeeService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using JobSeek.Api.Models.Entities;
+using JobSeek.Api.Models.Input;
+using JobSeek.Api.Repository.Contracts;
+using JobSeek.Api.Validations;
+using Mapster;
+using System.Text.RegularExpressions;
+
+namespace JobSeek.Api.Services.Implementation
+{
+    public class EmployeeService : BaseService<Employee, EmployeeInput>
+    {
+        private readonly IBaseRepository<Employee> _repository;
+
+        public EmployeeService(IBaseRepository<Employee> repository) : base(repository)
+        {
+            _repository = repository;
+        }
+        public Employee Create(EmployeeInput input)
+        {
+            
+            //ckeck birthday
+            if (input.BirthDate <= DateTimeOffset.Now)
```

Comment:
this is not correct comparation, here you write if the inputted birthdate is smaller than now, then throw an exception, which is not correct , and you should do this comparation in validation attribute